pkg/asset/manifests/featuregate.go

```diff
@@ -0,0 +1,77 @@
+package manifests
+
+import (
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	featureGateFilename = filepath.Join(manifestDir, "cluster-featuregate-02-config.yml")
+)
+
+// FeatureGate generates the cluster-ingress-*.yml files.
+type FeatureGate struct {
+	FileList []*asset.File
+}
+
+var _ asset.WritableAsset = (*FeatureGate)(nil)
+
+// Name returns a human friendly name for the asset.
+func (*FeatureGate) Name() string {
+	return "FeatureGate Config"
+}
+
+// Dependencies returns all of the dependencies directly needed to generate the asset.
+func (*FeatureGate) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.InstallConfig{},
```

Comment:
The InstallConfig dependency is unnecessary.

Reply:
> The InstallConfig dependency is unnecessary.

removing it appears to have caused multiple failures.

Reply:
I should have been more complete with my comment. You need to also remove the code in `Generate` where the InstallConfig is retrieved.

https://github.com/openshift/installer/blob/5b62a4df18af00285191b512b3965fd55fb3611c/pkg/asset/manifests/featuregate.go#L38-L39

Reply:
> I should have been more complete with my comment. You need to also remove the code in `Generate` where the InstallConfig is retrieved.

and now I feel slow.